tools/upload_text.php

```diff
@@ -221,6 +221,11 @@
     echo "<h1>$title</h1>";
     echo "<h2>", sprintf("Project: %s", $project->nameofwork), "</h2>";
 
+    // Disable gzip compression so we can flush the buffer after each step
+    // in the process to give the user some progress details. Note that this
+    // doesn't necessarily work for all browsers.
+    apache_setenv('no-gzip', '1');
```

Comment:
I think this needs to be set before we start outputting any text, otherwise Apache will have already enabled gzip for the buffer. That's why I suggested you set this in `get_upload_args()` which is almost guaranteed to be called before anything is output.

Reply:
I've moved it up above the slim_header()
Does it only affect an individual visit to the server? If so then it would have to be in this section of the code where the header with `get_upload_args()` is not executed.
I've moved it in remote_file_manager also.

Reply:
Ah, good point. Moving it before `slim_header()` works fine.